anyrun.json

```diff
@@ -254,6 +465,361 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+        {
+            "action": "detonate url",
+            "description": "Load a URL for analysis and returns the ID of created task",
+            "verbose": "For the 'obj_url' parameter Size range: 5-512. For the 'opt_timeout' Size range: 10-660. For the 'obj_ext_cmd' Size range: 2-256. For the 'obj_ext_useragent' Size range: 2-256.",
```

Comment:
similar comment like the one above, Should we include the range info in the description of the params as well?
This would ensure those restriction shows up when the users update those values.

Reply:
Fixed